mnd/mnd/integration/mims/mims_service.py

```diff
@@ -60,7 +60,7 @@ def mims_product_details(product):
             mims = ", ".join(product_details['mimsClasses'])
             name = product_details.get('productName', '')
 
-            active_ingredient = next((p.activeIngredient for p in mims_product_iter(name) if p.id == product), None)
+            active_ingredient = next(iter(product_details["acgs"]), {'acgName': None})['acgName']
```

Comment:
Is `acgName` guaranteed to exist on the product detail object?

Reply:
Not clear from the docs, but can't hurt
![image](https://user-images.githubusercontent.com/17246877/198436581-30898210-d418-4095-80ea-cc6431988be6.png)